src/javascript/_common/base/login.js

```diff
@@ -8,20 +8,21 @@ const removeCookies       = require('../storage').removeCookies;
 const urlForCurrentDomain = require('../url').urlForCurrentDomain;
 const isLoginPages        = require('../utility').isLoginPages;
 const TrafficSource       = require('../../app/common/traffic_source');
+const urlFor = require('../language').urlFor;
```

Comment:
Could you add more spaces before equals sign to match it with other equal signs above?

Reply:
Thanks for noticing, forget about that one. Changed.